.travis.yml

```diff
@@ -58,6 +58,7 @@ deploy:
     email: stefan.penner+ember-cli@gmail.com
     api_key:
       secure: PQRsq4JcFYPXOqov1jwPw63WnwYYpyKby/2akUjq8PHwwgCDEt34Z8yqQdJogqKmKRjijsiWu+b54fbIpZ6TMvqPL18F8FbCl6e2zoAY55VUmmVkx9wkvuPydTDs2fuPqk6LFvmIMiq+pjEPlSeWg3GpFSkFjfs0WZlO33xMyWw=
+    skip_cleanup: true
```

Comment:
With this added, and with using the --skip-cleanup option to ember-try, I think (assuming travis behaves as I am understanding) you'll deploy/publish the package.json as it was when switched to the last scenario run. Right now this is `ember-default` which should have no changes, but seems very coincidental.

Reply:
good point :(